src/plugins/maps_legacy/public/map/opensearch_dashboards_map.js

```diff
@@ -41,6 +41,7 @@ import { i18n } from '@osd/i18n';
 import { ORIGIN } from '../common/constants/origin';
 import { getToasts } from '../opensearch_dashboards_services';
 import { L } from '../leaflet';
+import './_legend.scss';
```

Comment:
Shouldn't this already be imported by `src/plugins/maps_legacy/public/map/index.scss`?

Reply:
Thank you, @BSFishy for your review. I have removed the redundant import in the latest commit